source/darwin/securityframework_ecc.c

```diff
@@ -332,6 +335,7 @@ struct aws_ecc_key_pair *aws_ecc_key_pair_new_from_public_key(
     return NULL;
 }
 
+#if !defined(AWS_OS_IOS)
 struct aws_ecc_key_pair *aws_ecc_key_pair_new_generate_random(
```

Comment:
should we ifdef out the declaration in the H file too?

Reply:
YEs. Fixed.